Buildings/Controls/OBC/FDE/PackagedRTUs/Validation/OperatingModeSimulation.mos

```diff
@@ -0,0 +1,29 @@
+// Script generated by Dymola Mon Jul 13 13:30:55 2020
```

Comment:
delete this file: `Buildings/Controls/OBC/FDE/PackagedRTUs/Validation/OperatingModeSimulation.mos `

Reply:
Agree